bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/java/org/openhab/binding/modbus/helioseasycontrols/internal/HeliosEasyControlsHandler.java

```diff
@@ -339,8 +339,11 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     try {
                         writeValue(channelId, v);
                         if (variableMap != null) {
-                            updateState(variableMap.get(channelId), v);
-                            updateStatus(ThingStatus.ONLINE);
+                            HeliosVariable variable = variableMap.get(channelId);
+                            if (variable != null) {
+                                updateState(variable, v);
+                                updateStatus(ThingStatus.ONLINE);
+                            }
```

Comment:
I was thinking about that. But finally decided to put it in the if-clause because I think that a null value here is some sort of error state and the thing should probably not be ONLINE.